src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs

```diff
@@ -501,8 +497,8 @@ internal static class ConsoleInputWithNativeMethods
             /// <returns>True if it was successful in adding all characters to console input buffer.</returns>
             internal static bool AddToConsoleInputBuffer(string str, bool newLine)
             {
-                IntPtr handle = ConsoleInputWithNativeMethods.GetStdHandle(ConsoleInputWithNativeMethods.STD_INPUT_HANDLE);
-                if (handle == IntPtr.Zero)
+                SafeFileHandle handle = Interop.Windows.GetStdHandle(Interop.Windows.STD_INPUT_HANDLE);
```

Comment:
It is used only in the method.
```suggestion
                using SafeFileHandle handle = Interop.Windows.GetStdHandle(Interop.Windows.STD_INPUT_HANDLE);
```